lib/opencensus/span.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Opencensus.Span do
+  @moduledoc """
+  Elixir convenience translation of `:opencensus.span`.
+
+  Most likely to be of use while writing unit tests, or packages that deal with spans.
+  Less likely to be of use while writing application code.
+  """
+
+  alias Opencensus.SpanContext
+
+  require Record
+  @fields Record.extract(:span, from_lib: "opencensus/include/opencensus.hrl")
+  Record.defrecordp(:span, @fields)
+
+  defstruct Keyword.keys(@fields)
+
+  @doc "Get a span struct given a record."
+  @spec from(:opencensus.span()) :: %__MODULE__{}
+  def from(record) when Record.is_record(record, :span), do: struct!(__MODULE__, span(record))
```

Comment:
As there is `from/1` function, there should be also `to/1` function that would change struct back to record, for interoperability purposes.